app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp_command, source_file, copied_file = command.split()
```

Comment:
It's preferable to use a more descriptive variable name instead of `cp_command`. For example, you could use `command_name`.